client/hosting/monitoring/components/site-monitoring.tsx

```diff
@@ -232,7 +231,14 @@ export interface HTTPCodeSerie {
 }
 
 function colorToAlpha( color: keyof typeof colorStudio.colors, alpha: number ) {
-	return chroma( colorStudio.colors[ color ] ).alpha( alpha ).hex();
+	const hex = colorStudio.colors[ color ];
```

Comment:
Let's add a guard if the user inserts a value over 1 to return the original color.

```suggestion
	if ( alpha < 0 || alpha >= 1 ) {
		return colorStudio.colors[ color ];
	}
	const hex = colorStudio.colors[ color ];
```

Reply:
Good call! I've added this in [a312a16](https://github.com/Automattic/wp-calypso/pull/95614/commits/a312a167c24e0e2d2d768b753942a678d0912b92)